src/components/sbb-link/sbb-link.tsx

```diff
@@ -63,88 +76,134 @@ export class SbbLink {
   @Prop() public textSize: InterfaceLinkAttributes['textSize'] = 's';
 
   /**
-   * Choose the link style variant.
+   * Applies link inline styles (underline, inherit coloring/font-size etc).
    */
-  @Prop() public variant: InterfaceLinkAttributes['variant'] = 'block';
+  @Prop() public inline = false;
```

Comment:
This should still be variant (with `block` and `inline`)?

Reply:
I'd prefer to use `variant: 'block' | 'inline'` instead of a boolean